src/components/token/TokenAutoCompleteField.js

```diff
@@ -146,7 +184,7 @@ class TokenAutoCompleteField extends React.Component {
         value={this.state.searchText}
         onKeyUp={this.onSearchTextKeyUp}
         onChange={this.onSearchTextChanged}
-        placeholder="Search UID, name or symbol"
+        placeholder="Hathor (HTR) - Type to search for other tokens by UID, name or symbol"
         aria-label="Search"
         ref={ref => {this.autoCompleteInputRef = ref}} />
```

Comment:
Do we use this for anything?

I know it's from previous PRs, but maybe we could remove it.

Reply:
Yeah, it was not used. Removed here https://github.com/HathorNetwork/hathor-explorer/pull/201/commits/1de07de4fef02b6d77ee93c568aed393d7b655df